src/lib/components/particles/square-cut-corner-icon/index.jsx

```diff
@@ -0,0 +1,29 @@
+import defaultStyles from "./style.module.scss"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+export const SquareCutCornerIcon = ({cornerColor, squareColor, squareSize, cornerPartyAbbreviation, squarePartyAbbreviation, styles}) => {
```

Comment:
You might not agree with me here, but I think it would be better to remove the `cornerPartyAbbreviation` and `squarePartyAbbreviation` props. The reason is that these props depend on a convention that exists outside of the context of this component. 

In practice this means that the component becomes more fragile, e.g. if we change the `fill-color-${abbreviation}` convention, then this change cascades down to this component and breaks it.

My proposed solution would be to pass the colors in the styles prop, i.e.

```js
const styles: {
 corner: `fill-color-${abbreviation}`,
 square: `fill-color-${abbreviation}`,
}

<SquareCutCornerIcon styles={styles} />
```

Reply:
Good to know. Yes happy to do this. Will look at some other components to figure out how to do this more fully